src/main/resources/templates/modify.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th="https://www.thymeleaf.org">
+<head>
+    <title>Modification</title>
+</head>
+<body>
+<img src="/images/logo.jpg" style="float: left; margin-right: 20px; width: 250px;" />
+<!--  <p th:if="${showErrorMessage}">Incorrect login!</p>-->
+<h1>User modify</h1>
+<form action="#" method="post" th:action="@{/user/modify}" th:object="${user}">
+
+    <input th:field="*{id}" type="hidden" placeholder="Id"/><br/>
+    <p th:errors="*{lastName}">first Name errors</p>
+    <input th:field="*{lastName}" type="text" placeholder="Last name"/><br/>
+    <p th:errors="*{firstName}">last Name errors</p>
+    <input th:field="*{firstName}" type="text" placeholder="First name"/><br/>
+    <p th:errors="*{dateOfBirth}">dateOfBirth errors</p>
+    <input th:field="*{dateOfBirth}" type="date" placeholder="Date of Birth"/><br/>
+    <p th:errors="*{email}">email errors</p>
+    <input th:field="*{email}" type="email" placeholder="E-mail"/><br/>
+<!--    <p th:errors="*{isNewsletter}">is Newsletter errors</p>-->
```

Comment:
Żeby to naprawić - zmień nazwę pola z isNewsletter na przykładowo "newsletter" i zaktualizuj odpowiednio wszystkie referencje do tego pola